applications/asset_tracker_v2/src/modules/Kconfig.cloud_module

```diff
@@ -36,6 +36,9 @@ choice NRF_CLOUD_CLIENT_ID_SRC
 	default NRF_CLOUD_CLIENT_ID_SRC_RUNTIME
 endchoice
 
+config NRF_CLOUD_CELL_POS
```

Comment:
Please document this.

Reply:
This is a re-definition of the original kconfig, and here it's not user configurable. So I don't think it makes sense to document it again here.